include/amici/abstract_model.h

```diff
@@ -82,6 +82,22 @@ class AbstractModel {
                     const AmiVector &dx, const AmiVector &xdot,
                     SUNMatrix J) = 0;
 
+    /**
+     * @brief Dense Jacobian function
+     * @param t time
+     * @param cj scaling factor (inverse of timestep, DAE only)
+     * @param x state
+     * @param dx time derivative of state (DAE only)
+     * @param xB Vector with the adjoint states
+     * @param dxB Vector with the adjoint derivative states
+     * @param xBdot Vector with the adjoint right hand side (unused)
+     * @param JB dense matrix to which values of the jacobian will be written
+     */
+    virtual void fJB(const realtype t, realtype cj, const AmiVector &x,
```

Comment:
I think this is what causes the CI errors...
Not yet sure what the problem is precisely. Will have a closer look tomorrow

Reply:
Done @paszkow . If you accept my pull request into your fork, then the CI errors should be fixed now.

Reply:
Sure I will. I already have all your comments implemented. Need to double-check the correctness of the results I am getting before pushing.

Reply:
Would be great if you could add these tests as unittests for the proposed, if you need help with that, please let me know!

Reply:
Sounds like a good idea. Will do.

Reply:
> Sure I will. I already have all your comments implemented. Need to double-check the correctness of the results I am getting before pushing.

At least for the small steady-state example problem, things looked correct when I tested them...